lib/compilers/pascal-win.ts

```diff
@@ -117,10 +116,7 @@ export class PascalWinCompiler extends BaseCompiler {
         await fs.writeFile(
             filename,
             // prettier-ignore
-            'program prog;\n' +
-            'uses ' + unitName + ' in \'' + unitPath + '\';\n' +
-            'begin\n' +
-            'end.\n',
+            `program prog;\nuses ${unitName} in \'${unitPath}\';\nbegin\nend.\n`,
```

Comment:
This was prefixed with `// prettier-ignore`, is there an equivalent for biome? If not please remove the comment